project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.5.7
+sbt.version=1.7.3
```

Comment:
Latest version is `1.8.2` which I think is worth updating to

Reply:
wow that was a quick response 😁